src/tls_wrap.cc

```diff
@@ -1033,6 +1033,11 @@ int TLSWrap::SelectSNIContextCallback(SSL* s, int* ad, void* arg) {
   Local<Object> object = p->object();
   Local<Value> ctx;
 
+  // Set the servername as early as possible
+  Local<Object> owner = p->GetOwner();
+  USE(owner->Set(env->context(), env->servername_string(),
+      OneByteString(env->isolate(), servername)));
```

Comment:
Tiny style nit: can you line up the arguments?

I'm a little ambivalent about discarding the return value with `USE(...)`. I'm inclined to say it should `return SSL_TLSEXT_ERR_NOACK` when setting the property fails.

Reply:
> Tiny style nit: can you line up the arguments?

Done.

> I'm a little ambivalent about discarding the return value with USE(...). I'm inclined to say it should return SSL_TLSEXT_ERR_NOACK when setting the property fails.

Good reminding! I think returning `SSL_TLSEXT_ERR_NOACK` is okay as the `owner` looks abnormal in this scenario.